bimsb/packages/bioinformatics-nonfree.scm

```diff
@@ -1882,3 +1883,95 @@ MACE is a bioinformatics tool dedicated to analyze ChIP-exo data.")
     ;; others say "free for all use", others say "GPL version 1.3" (?), yet
     ;; others say "Artistic license".
     (license license:gpl2+)))
+
+;; Although this program is released under the GPL it depends on
+;; ViennaRNA and bowtie1, which are non-free software.
+(define-public mirdeep2
+  (package
+    (name "mirdeep2")
+    (version "0.0.92")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                   (url "https://github.com/rajewsky-lab/mirdeep2.git")
+                   (commit (string-append "v" version))))
+              (file-name (string-append name "-" version "-checkout"))
+              (sha256
+               (base32
+                "1gasx6vb38n4vbjbn5s7ch2rpflgf2crdd8mdp2xm571w2j01wmm"))))
+    (build-system gnu-build-system)
+    (arguments
+     `(#:tests? #f ;; no check target
+       #:phases
+       (modify-phases %standard-phases
+         (delete 'configure)
+         (replace 'build
+           (lambda* (#:key inputs outputs #:allow-other-keys)
+             (let ((out (assoc-ref outputs "out")))
+               ;; patch scripts checking for ../install_successful file
+               (for-each
+                 (lambda (script)
+                   (substitute* script
+                     (("\\$bn/../install_successful")
+                      (string-append out
+                                     "/share/mirdeep2/install_successful"))))
+                 '("src/mapper.pl"
+                   "src/miRDeep2.pl"
+                   "src/quantifier.pl"))
+               ;; patch script using ../Rfam_for_miRDeep.fa file
+               (substitute* "src/miRDeep2.pl"
+                 (("\\$\\{scripts\\}/../Rfam_for_miRDeep.fa")
```

Comment:
Are these properly escaped?

Reply:
I'll escape them just in case when finishing this PR.

Reply:
done in https://github.com/BIMSBbioinfo/guix-bimsb-nonfree/pull/1/commits/2ccb7351e01e5bf94aab8bdf219ad160f3007dc6